.github/SOURCE_CONTRIBUTING.md

```diff
@@ -1,36 +1,47 @@
 **Thank you for taking the time to contribute to the development of WildBeast!**
+
 Please follow these rules when making contributions to this repository.
 
 # Contribution rules
+
 ## Unwanted contributions
+
 1. Style violations fixes
 2. ESLint rules changes
 3. Alias additions (Only applies if no new commands are provided)
-4. Issues stemming from usage of normal user accounts instead of OAuth bot accounts
```

Comment:
Why is this removed?

Reply:
We dropped support for normal user accounts in commit 6ac89ec on April 30, 2017.

Reply:
From that commit we dropped the ability to use email/password to login, but that doesn't take away that people could still use user account tokens to connect.

Reply:
True. However, I introduced this line myself in 2016 in commit acebebc, namely referring to the usage of the user and password login feature, a feature that was dropped in 6ac89ec.

As for whether it should stay around or not, I think that in the current Discord ecosystem not running bots under a normal user account is expected, and thus it is not necessary for us to state that explicitly.

Reply:
Then isn't that warning sufficient enough? I don't see why that should be in the contribution guidelines in the first place either (Which is on me, I fully acknowledge I'm part of the problem lol) - if anything it concerns end users. The line was not in any sort of issue template, and on Discord it's not really necessary to state it as previously mentioned.